packages/mongo/mongo_livedata_tests.js

```diff
@@ -3229,18 +3229,6 @@ Meteor.isServer && testAsyncMulti("mongo-livedata - update with replace forbidde
   }
 ]);
 
```

Comment:
Hey @radekmie, I remove those two tests. They were the reason why Travis CI was failing. But I'd like to confirm with you if that is ok. From what I could understand, the connection behavior now changed, and we fail silently when we cannot connect to MongoDB.
I think the first one is ok to remove, but I'm not sure about the second. 

cc @Grubba27

Reply:
So, it looks like this:
1. `mongo-livedata - connection failure throws` was a test that we already had previously, and I only changed it to reflect the functional change. It is not failing silently now, as we await the connection promise in `Meteor.startup`, just like @zodern suggested.
2. `mongo-livedata - oplog - _onFailover` is a test I added, as there's no other means to test the `_onFailover` functionality.

Now, while I can imagine the latter being problematic, as it performs an actual failover of the database, the former should be fine good to stay, I think.

_(If possible, let's keep both, but just comment them out or something.)_

Reply:
Oh, I see, I think it was changed to fail silently, but then you changed it back. Okay, I'll leave them commented for now. I am also pinging @filipenevola here so he can help us.